extensions/query-store/src/reports/baseQueryStoreReport.ts

```diff
@@ -15,7 +15,14 @@ export abstract class BaseQueryStoreReport {
 	protected configureDialog?: ConfigureDialog;
 	protected configureButton?: azdata.ButtonComponent;
 
-	constructor(private reportTitle: string, protected resizeable: boolean, private extensionContext: vscode.ExtensionContext) { }
+	/**
+	 * Constructor
+	 * @param reportTitle Title of report shown in toolbar
+	 * @param reportId Id of tab used in query store dashboard
+	 * @param resizeable Whether or not the sections of the report are resizeable
+	 * @param extensionContext
+	 */
+	constructor(private reportTitle: string, private reportId: string, protected resizeable: boolean, private extensionContext: vscode.ExtensionContext) { }
```

Comment:
nit: IIRC, we generally name the private variables starting with _ and then name (eg. _reportTitle).

Reply:
I think that's mostly Alan.  I intentionally try to avoid using leading underscores in variable names unless they're discards.

Reply:
it's inconsistent throughout the ADS codebase. I tend to not use them since they make doc comments harder to read if they're declared in the constructor like this. I'll leave these as is for now, but we can switch them all later if that's easier to read.

Reply:
Makes sense. I will follow that too in future 👍 .